tracking_model/mixins.py

```diff
@@ -12,11 +12,24 @@ def __init__(self, instance):
 class TrackingModelMixin(object):
 
     TRACKED_FIELDS = None
+    tracker_class = Tracker
```

Comment:
I _imagine_ it might convenient to be able to configure the same `Tracker` class project-wide, rather than setting this attribute on each models that subclass `TrackingModelMixin`.

With that in mind, I would suggest a Django setting `TRACKING_MODEL_TRACKER_CLASS` (or something along these lines) to set an import path for a class to use everywhere, whihc value would default to `"tracking_model.Tracker"`.

However, it does make the implementation a bit more complicated. A keyword here is "imagine": it's not a problem I faced, just something I think I might need.

Reply:
> able to configure the same Tracker class project-wide

For project-wide usage, I thought that one could easily subclass `TrackingModelMixin` and override the `tracking_class` and use it everywhere. (Took DRF's approach here)

But yeah, from the end user perspective, this settings approach would completely eliminate the overriding.

Reply:
That's actually a great solution, much simpler and direct than a setting.